pyvisa_py/tcpip.py

```diff
@@ -68,15 +265,18 @@ def list_resources() -> List[str]:
         return []
 
     def after_parsing(self) -> None:
-        # TODO: board_number not handled
```

Comment:
This is still true and should be preserved.

Reply:
Agree, deleting it was an unintentional error on my part.